src/main/java/io/github/railroad/menu/TopMenu.java

```diff
@@ -0,0 +1,238 @@
+
+        fileMenu.getItems().addAll(open, recent, new SeparatorMenuItem(), save, saveAs, export);
+    }
+
+    /**
+     * Creates the new file submenu and all of its submenus.
+     *
+     * @param fileMenu the object that contains the new file submenu.
+     * @author TheOnlyTails
+     */
+    public void createFileNewMenu(Menu fileMenu) {
+        final var newMenu = new Menu(getLocalization("menu.file.new", ENGLISH));
+        final var javaProject = makeMenuItem(getLocalization("menu.file.new.java_project", ENGLISH))
+                .graphic(createImage("/assets/img/java_project.png"))
+                .get();
+
+        final var project = makeMenuItem(getLocalization("menu.file.new.project", ENGLISH))
+                .graphic(createImage("/assets/img/project.png"))
+                .get();
+
+        final var workingSet = makeMenuItem(getLocalization("menu.file.new.java_working_set", ENGLISH))
+                .graphic(createImage("/assets/img/working_set.png"))
+                .get();
+
+        final var packageItem = makeMenuItem(getLocalization("menu.file.new.package", ENGLISH))
+                .graphic(createImage("/assets/img/package.png"))
+                .get();
+
+        final var source = makeMenuItem(getLocalization("menu.file.new.source_folder", ENGLISH))
+                .graphic(createImage("/assets/img/source_folder.png"))
+                .get();
+
+        final var file = makeMenuItem(getLocalization("menu.file.new.file", ENGLISH))
+                .graphic(createImage("/assets/img/file.png"))
+                .action(event -> createNewGenericFile())
+                .get();
+
+        final var folderItem = makeMenuItem(getLocalization("menu.file.new.folder", ENGLISH))
+                .graphic(createImage("/assets/img/folder.png"))
+                .get();
+
+        final var classItem = makeMenuItem(getLocalization("menu.file.new.class", ENGLISH))
+                .graphic(createImage("/assets/img/class.png"))
+                .action(event -> createNewJavaFile(CLASS, CLASS_NAME))
+                .get();
+
+        final var interfaceItem = makeMenuItem(getLocalization("menu.file.new.interface", ENGLISH))
+                .graphic(createImage("/assets/img/interface.png"))
+                .action(event -> createNewJavaFile(INTERFACE, INTERFACE_NAME))
+                .get();
+
+        final var enumItem = makeMenuItem(getLocalization("menu.file.new.enum", ENGLISH))
+                .graphic(createImage("/assets/img/enum.png"))
+                .action(event -> createNewJavaFile(ENUM, ENUM_NAME))
+                .get();
+
+        final var annotationItem = makeMenuItem(getLocalization("menu.file.new.annotation", ENGLISH))
+                .graphic(createImage("/assets/img/annotation.png"))
+                .action(event -> createNewJavaFile(ANNOTATION, ANNOTATION_NAME))
+                .get();
+
+        newMenu.getItems().addAll(javaProject, project, workingSet, new SeparatorMenuItem(),
```

Comment:
You used `fileMenu.getItems().add(new SeparatorMenuItem());` above, but here you add a new separator in between other items, consider consistency here?

Reply:
The separator menu item is not a regular item: it's used to make partitions between other items. It's mostly a design element.